### PR TITLE
fix(apm): metric grouping addition

### PR DESCRIPTION
**Pull request by homelessbirds**
Addressing customer feedback

### CODE REVIEW

File: src/content/docs/new-relic-solutions/solve-common-issues/troubleshooting/metric-grouping-issues.mdx [hunk 1]
```diff
@@ -164,6 +164,10 @@ If the problem persists, follow the procedures for your agent:
 
 You can also edit and create metric normalization rules in the UI. For more details, see [Metric normalization](/docs/new-relic-solutions/new-relic-one/ui-data/metric-normalization-rules).
 
+<Callout variant="tip">
+If you want to group data that is reporting to the Metrics explorer, you can use [Metric normalization rules](/docs/new-relic-solutions/new-relic-one/ui-data/metric-normalization-rules/) to match those incoming metrics and replace those matched metrics.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/solve-common-issues/troubleshooting/metric-grouping-issues.mdx:168]:
```suggestion
If you want to group data that's reporting to the Metrics explorer, you can use [Metric normalization rules](/docs/new-relic-solutions/new-relic-one/ui-data/metric-normalization-rules/) to match those incoming metrics and replace those matched metrics.
```
replace those matched metrics with what?

**Review by paperclypse — APPROVED** [review]

**Comment by homelessbirds** [comment]
@paperclypse this was a draft and wasn't ready for review yet. I should have made a note of that. But thank you for taking the time to review it.

**Comment by austin-schaefer** [comment]
Hey @homelessbirds , is this PR still something you're actively working on?

**Comment by homelessbirds** [comment]
After a lot of back and forth with gts, decided that this addition isn't necessary. Closing.

**Pull request closed without merging**